webapp/containers/App/App.js

```diff
@@ -102,7 +109,7 @@ class App extends PureComponent {
 }
 
 const mapStateToProps = state => ({
-  sortedPluginMeta: plugins.metadataWithUniquePaths(state),
+  sidebarNavItems: nav.sortedSidebarItems(state),
```

Comment:
using the sidebar nav selector. by default the items stay in an unsorted state in the redux store. Using the selector you can get them sorted (and it will only recalculate if there's been a change in the state)